ktor-server/ktor-server-servlet/jvm/src/io/ktor/server/servlet/KtorServlet.kt

```diff
@@ -46,10 +46,17 @@ public abstract class KtorServlet : HttpServlet(), CoroutineScope {
      */
     protected abstract val upgrade: ServletUpgrade
 
-    override val coroutineContext: CoroutineContext =
+    /**
+     * Base context for call handling
+     */
+    protected abstract val parentCoroutineContext: CoroutineContext
```

Comment:
It's a breaking change: a new protected field in a public abstract class

Reply:
updated